pyfftw/interfaces/_utils.py

```diff
@@ -126,7 +131,4 @@ def _Xfftn(a, s, axes, overwrite_input, planner_effort,
         FFTW_object(input_array=a, output_array=output_array, 
                 normalise_idft=normalise_idft)
     
```

Comment:
In these cases, the transform is performed on a copy of the original array, so the original would not be clobbered.